provision.py

```diff
@@ -124,10 +130,20 @@ def main():
             "source %s\n" % (os.path.join(VENV_PATH, "bin", "activate"),),
         ])
 
-    # Switch current Python context to the virtualenv.
+    # Switch current Python context to the python3 virtualenv
+    activate_this = os.path.join(PY3_VENV_PATH, "bin", "activate_this.py")
+    execfile(activate_this, dict(__file__=activate_this))
+
+    run(["pip", "install", "--upgrade", "pip"])
+    # install requirement
+    run(["pip", "install", "--no-deps", "--requirement",
+         os.path.join(ZULIP_PATH, "py3_test_reqs.txt")])
+
+    # Switch current Python context to the python2 virtualenv.
     activate_this = os.path.join(VENV_PATH, "bin", "activate_this.py")
     execfile(activate_this, dict(__file__=activate_this))
 
+    run(["pip", "install", "--upgrade", "pip"])
```

Comment:
I added this line for consistency with corresponding line of python 3 virtualenv and so that I have to download less when I run provision.py multiple times because newer versions of pip cache downloaded packages.